source/MechEngineer.csproj

```diff
@@ -67,4 +67,13 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
   </ItemGroup>
+    <Target Name="CopyDLLs" AfterTargets="Build">
+    <Message Text="Executing CopyDLLs task" Importance="High" />
+    <PropertyGroup>
+      <CopyToDir>
+      </CopyToDir>
+    </PropertyGroup>
+    <Copy SourceFiles="$(TargetDir)$(ProjectName).dll" DestinationFolder="D:\Steam\steamapps\common\BATTLETECH\Mods\$(ProjectName)" />
+    <Message Text="Copied build files" Importance="High" />
```

Comment:
There is already a section to copy the dll to the indented folder

only thing missing would be to make it configurable as it defaults to $SolutionDir

Reply:
I've improved it, you can now override the destination using the ModDir property, see the Directory.Build.props.template on how to adjust your settings